modules/tinymce/src/core/main/ts/api/Options.ts

```diff
@@ -1029,6 +1034,10 @@ const getApiKey = option('api_key');
 const isDisabled = option('disabled');
 const getExtendedMathmlAttributes = option('extended_mathml_attributes');
 const getExtendedMathmlElements = option('extended_mathml_elements');
+const shouldHaveListFeatures = (editor: Editor): boolean => {
+  return editor.hasPlugin('lists') || editor.hasPlugin('advlist');
+};
+const shouldIndentOnTab = (editor: Editor): boolean => option('lists_indent_on_tab')(editor) && shouldHaveListFeatures(editor);
```

Comment:
It's a simple function, just inline it when this happens I think 😂 

```suggestion
const shouldIndentOnTab = (editor: Editor): boolean => editor.options.get('lists_indent_on_tab') && shouldHaveListFeatures(editor);
```